src/server/battlekards.js

```diff
@@ -125,17 +126,37 @@ exports.initialiseBattlekardsSocketIo = function initialiseBattlekardsSocketIo(i
         if (monsterIndex !== -1) {
           const monster = _.cloneDeep(game.players[game.playersTurn].hand[monsterIndex]);
           if (monster.type === 'monster') {
-            monster.canAttack = false;
-
-            game.players[playerId].hasSummoned = true; // Player cannot summon again
-            game.players[game.playersTurn].hand.splice(monsterIndex, 1);
-            game.players[game.playersTurn].monsters.push(monster);
+            if (position === 'attak') {
+              monster.canAttack = false;
+              monster.position = 'attak';
+              game.players[playerId].hasSummoned = true; // Player cannot summon again
+              game.players[game.playersTurn].hand.splice(monsterIndex, 1);
+              game.players[game.playersTurn].monsters.push(monster);
 
-            socket.emit('summoned', monster);
-            sockets[game.inactivePlayer].emit('opponentSummoned', {
-              monster,
-              opponentsHandSize: game.players[game.playersTurn].hand.length,
-            });
+              socket.emit('summoned', monster);
+              sockets[game.inactivePlayer].emit('opponentSummoned', {
+                monster,
+                opponentsHandSize: game.players[game.playersTurn].hand.length,
+                position,
+              });
+            } else {
+              monster.canAttack = false;
+              monster.position = 'defense';
+              game.players[playerId].hasSummoned = true; // Player cannot summon again
+              game.players[game.playersTurn].hand.splice(monsterIndex, 1);
+              game.players[game.playersTurn].monsters.push(monster);
+                  // Different here
+              socket.emit('summoned', monster);
+                  // Remove monster data for defense
+              monster.name = '';
+              monster.attributes.attack = '';
+              monster.attributes.defense = '';
+              sockets[game.inactivePlayer].emit('opponentSummoned', {
+                monster,
+                opponentsHandSize: game.players[game.playersTurn].hand.length,
+                position,
+              });
+            }
```

Comment:
There's a lot of duplicate code here. Could be refactored into something like:

```javascript
if (position === 'attak' || position === 'defense') {
  monster.canAttack = false;
  monster.position = position;
  game.players[playerId].hasSummoned = true; // Player cannot summon again
  game.players[game.playersTurn].hand.splice(monsterIndex, 1);
  game.players[game.playersTurn].monsters.push(monster);
  socket.emit('summoned', monster);
  if (position === 'defense') {
    monster.name = '';
    monster.attributes.attack = '';
    monster.attributes.defense = '';
  }
  sockets[game.inactivePlayer].emit('opponentSummoned', {
    monster,
    opponentsHandSize: game.players[game.playersTurn].hand.length,
    position,
  });
} else {
  socket.emit('invalidMove', 'position must be attack or defense');
}
```

Reply:
Actually, `delete` would be better than setting the attributes to empty strings:

```javascript
delete monster.name;
delete monster.attributes;
```